keanu-python/tests/test_tensor.py

```diff
@@ -81,11 +81,11 @@ def test_cannot_pass_generic_ndarray_to_Tensor(generic):
     assert str(excinfo.value) == "Generic types in an ndarray are not supported. Was given {}".format(type(generic))
 
 
-def test_cannot_pass_empty_ndarray_to_Tensor():
+def test_can_pass_empty_ndarray_to_Tensor():
     with pytest.raises(ValueError) as excinfo:
         Tensor(np.array([]))
 
-    assert str(excinfo.value) == "Cannot infer type because the ndarray is empty"
+    assert str(excinfo.value) == "Cannot infer type because array is empty"
```

Comment:
It doesn't throw a ValueError anymore does it? And if the array is empty, I guess a Java exception would be thrown?

Reply:
it still throws a different `ValueError` - from `__infer_class_from_array`: https://github.com/improbable-research/keanu/blob/48762e3c27463b2b4b0759e2c1cd7ebcb192fc88/keanu-python/keanu/context.py#L86